ui/src/components/Dashboard/DocsSidebarList.jsx

```diff
@@ -142,8 +142,7 @@ export default class DocsSidebarList extends Component {
         <Typography component="span">(</Typography>
         {nodes.map((node, idx) => {
           const href = removeReadmeFromPath(`${DOCS_PATH_PREFIX}/${node.path}`);
-          const isLinkActive = removeReadmeFromPath(location.pathname) === href;
-
+          const isLinkActive = removeReadmeFromPath(location.pathname) === href || removeReadmeFromPath(location.pathname) === href + "/";
```

Comment:
@ruchikabgosain This should work but we can simplify it by removing the trailing slashes from the path.

```diff
--- a/ui/src/utils/removeReadmeFromPath.js
+++ b/ui/src/utils/removeReadmeFromPath.js
@@ -1 +1,2 @@
-export default (path = '') => path.replace(/\/?README\/?/, '');
+export default (path = '') =>
+  path.replace(/\/+$/, '').replace(/\/?README\/?/, '');
```

What do you think?

Reply:
I think that makes a lot of sense :)

Reply:
Hey @arku , I suppose you want me to directly update the <removeReadmeFromPath.js> and restore the changes for this <DocsSidebarList.jsx> file? 
I have updated it according to your changes, and i understand how this offers more clarity. 
Thankyou for your inputs :) Hope this resolves!